languages/translations/hu.json

```diff
@@ -0,0 +1,493 @@
+                    "destination": "Hajtson be {rotary_name:article} körforgalomba, majd hajtson ki {destination:article} irányába"
+                },
+                "exit": {
+                    "default": "Hajtson be a körforgalomba, majd hajtson ki {exit_number:article} kijáraton",
+                    "name": "Hajtson be a körforgalomba, majd hajtson ki {exit_number:article} kijáraton {way_name:article} felé",
+                    "destination": "Hajtson be a körforgalomba, majd hajtson ki {exit_number:article} kijáraton {destination:article} irányába"
+                },
+                "name_exit": {
+                    "default": "Hajtson be {rotary_name:article} körforgalomba, majd hajtson ki {exit_number:article} kijáraton",
+                    "name": "Hajtson be {rotary_name:article} körforgalomba, majd hajtson ki {exit_number:article} kijáraton {way_name:article} felé",
+                    "destination": "Hajtson be {rotary_name:article} körforgalomba, majd hajtson ki {exit_number:article} kijáraton {destination:article} irányába"
+                }
+            }
+        },
+        "roundabout": {
+            "default": {
+                "exit": {
+                    "default": "Hajtson be a körforgalomba, majd hajtson ki {exit_number:article} kijáraton",
+                    "name": "Hajtson be a körforgalomba, majd hajtson ki {exit_number:article} kijáraton {way_name:article} felé",
+                    "destination": "Hajtson be a körforgalomba, majd hajtson ki {exit_number:article} kijáraton {destination:article} irányába"
+                },
+                "default": {
+                    "default": "Hajtson be a körforgalomba",
+                    "name": "Hajtson a körforgalomba, majd hagyja el a körforgalmat {way_name:article} felé",
+                    "destination": "Hajtson a körforgalomba, majd hagyja el a körforgalmat {destination:article} irányába"
+                }
+            }
+        },
+        "roundabout turn": {
+            "default": {
+                "default": "Forduljon {modifier}",
+                "name": "Forduljon be {modifier} {way_name:article} felé",
+                "destination": "Forduljon {modifier} {destination:article} irányába"
+            },
+            "left": {
+                "default": "Forduljon balra",
+                "name": "Forduljon be balra {way_name:article} irányába",
+                "destination": "Forduljon balra {destination:article} irányába"
+            },
+            "right": {
+                "default": "Forduljon jobbra",
+                "name": "Forduljon be jobbra {way_name:article} felé",
+                "destination": "Forduljon jobbra {destination:article} irányába"
+            },
+            "straight": {
+                "default": "Hajtson egyenesen",
+                "name": "Hajtson egyenesen {way_name:article} felé",
+                "destination": "Hajtson egyenesen {destination} irányba"
```

Comment:
@kiskunk, maybe there should be `irányába` not `irányba`? And article `a(z)` is missing here?